rust/connlib/clients/apple/build.rs

```diff
@@ -1,145 +1,15 @@
-            "release"
-        } else {
-            "debug"
-        });
-        Self {
-            swift_pkg_dir,
-            swift_src_dir,
-            bridge_dst_dir,
-            swift_built_lib_dir,
-            release,
-            triple,
-            sdk,
-        }
-    }
-}
-
-fn gen_bridges(env: &Env) {
-    for path in BRIDGE_SRCS {
-        println!("cargo:rerun-if-changed={path}");
-    }
-    swift_bridge_build::parse_bridges(BRIDGE_SRCS)
-        .write_all_concatenated(&env.bridge_dst_dir, env!("CARGO_PKG_NAME"));
-}
-
-// We use `swiftc` instead of SwiftPM/`swift build` because of this limitation:
-// https://github.com/apple/swift-package-manager/pull/6572
-fn compile_swift(env: &Env) -> anyhow::Result<()> {
-    let swift_sdk = diva::Command::parse("xcrun --show-sdk-path --sdk")
-        .with_arg(env.sdk)
-        .run_and_wait_for_trimmed()?;
-    let swift_src_files = WalkDir::new(&env.swift_src_dir)
-        .into_iter()
-        .filter_map(Result::ok)
-        .filter_map(|entry| {
-            (entry.path().extension() == Some("swift".as_ref())).then(|| entry.path().to_owned())
-        });
-    std::fs::create_dir_all(&env.swift_built_lib_dir)?;
-    diva::Command::parse("swiftc -emit-library -static")
-        .with_args(["-module-name", SWIFT_PKG_NAME])
-        .with_arg("-import-objc-header")
-        .with_arg(env.swift_src_dir.join(BRIDGING_HEADER))
-        .with_arg("-sdk")
-        .with_arg(swift_sdk)
-        .with_args(["-target", &env.triple])
-        // https://github.com/apple/swift-package-manager/blob/55006dce81ae70cd8f2b78479038423eeebde1e4/Documentation/Usage.md#setting-the-build-configuration
-        .with_parsed_args(if !env.release {
-            "-Onone -g -enable-testing"
-        } else {
-            "-O -whole-module-optimization"
-        })
-        .with_arg("-o")
-        .with_arg(env.swift_built_lib_dir.join(SWIFT_LIB_NAME))
-        .with_args(swift_src_files)
-        .with_cwd(&env.swift_pkg_dir)
-        .run_and_wait()?;
-    Ok(())
-}
-
-fn link_swift(env: &Env) {
-    println!("cargo:rustc-link-lib=static={SWIFT_PKG_NAME}");
-    println!(
-        "cargo:rustc-link-search={}",
-        env.swift_built_lib_dir.display()
-    );
-    let xcode_path = diva::Command::parse("xcode-select --print-path")
-        .run_and_wait_for_trimmed()
-        .unwrap_or_else(|_| "/Applications/Xcode.app/Contents/Developer".to_owned());
-    println!("cargo:rustc-link-search={xcode_path}/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/macosx/");
-    println!("cargo:rustc-link-search=/usr/lib/swift");
-}
-
-fn main() -> anyhow::Result<()> {
-    // Early exit build script to avoid errors on non-Apple platforms.
-    if std::env::var("CARGO_CFG_TARGET_VENDOR").as_deref() != Ok("apple") {
-        return Ok(());
+    let bridges = vec!["src/lib.rs"];
+    for path in &bridges {
+        println!("cargo:rerun-if-changed={}", path);
```

Comment:
```suggestion
        println!("cargo:rerun-if-changed={path}");
```

Reply:
This wasn't a clippy error, so not changing it.